src/main/java/core/basesyntax/service/ProcessService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ProcessService {
+    void processData(List<FruitTransaction> fruitTransactionList,
+                     OperationStrategy operationStrategy);

```

Comment:
```suggestion
    void processData(List<FruitTransaction> fruitTransactionList);
```
pass OperationStrategy using constructor